scripts/update-lobster.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# this script pulls the evergreen-ci/lobster repo and copies the files so that
+# they can be served by evergreen
+# it should only be run from the base evergreen directory
+if [ -n "$LOBSTER_TEMP_DIR" ] && [ -n "$LOBSTER_TEMP_DIR" ]; then
```

Comment:
Rather than wrap this in a large, if can we just `exit 1` when `LOBSTER_TEMP_DIR` and `EVG_HOME` are unset? Like:
```sh
[[ -z "${LOBSTER_TEMP_DIR}" ]] && echo "missing environment variable LOBSTER_TEMP_DIR" && exit 1
[[ -z "${EVGHOME}" ]] && echo "missing environment variable EVGHOME" && exit 1
```